### PR TITLE
Add Tanzania CEO data

### PR DESCRIPTION
**Pull request by adebowaledaniel**
This is coming from the Tanzania North crop mask [task](https://github.com/nasaharvest/crop-mask/issues/202#issue-1321452665) assigned to me by @ivanzvonkov. 
After reviewing the labels (set 1 and 2) on CEO with some corrections, I export the sample labels to be added to the cropharvest dataset. 

PS: If there exists Tanzania CEO data on cropharvest, I would consider this an update.

### CODE REVIEW

**Comment by adebowaledaniel** [comment]
Check failed for no obvious reason. @gabrieltseng, what am I missing, please?

**Comment by gabrieltseng** [comment]
> Check failed for no obvious reason. @gabrieltseng, what am I missing, please?

@adebowaledaniel try merging main and rerunning the tests

File: process_labels/loading_funcs/tanzania.py [hunk 1]
```diff
@@ -100,6 +102,37 @@ def load_tanzania():
...
+    df = df.groupby("plotid").agg(
+        {
+            RequiredColumns.LON: "first",
+            RequiredColumns.LAT: "first",
+            RequiredColumns.GEOMETRY: "first",
+            "is_crop_mean": "mean",
+        }
+    )
+
+    df[RequiredColumns.COLLECTION_DATE] = datetime(2022, 8, 8)
```
Comment (gabrieltseng) [inline process_labels/loading_funcs/tanzania.py:128]: The filename says 2019, but the collection date says 2022; which is the correct value?

File: process_labels/loading_funcs/tanzania.py [hunk 2]
```diff
@@ -100,6 +102,37 @@ def load_tanzania():
...
+        {
+            RequiredColumns.LON: "first",
+            RequiredColumns.LAT: "first",
+            RequiredColumns.GEOMETRY: "first",
+            "is_crop_mean": "mean",
+        }
+    )
+
+    df[RequiredColumns.COLLECTION_DATE] = datetime(2019, 1, 2)
+    df[RequiredColumns.EXPORT_END_DATE] = datetime(2022, EXPORT_END_MONTH, EXPORT_END_DAY)
```
Comment (gabrieltseng) [inline process_labels/loading_funcs/tanzania.py:129]: @adebowaledaniel is this also supposed to be 2022?

Comment (adebowaledaniel) [reply process_labels/loading_funcs/tanzania.py:129]: Yes, the collection year is meant to be 2019! It's an updated Tanzania CEO 2019 dataset. Check the pull request description.

Comment (gabrieltseng) [reply process_labels/loading_funcs/tanzania.py:129]: yes but the export end date is 2022; should it be 2019?

Comment (adebowaledaniel) [reply process_labels/loading_funcs/tanzania.py:129]: Okay, I get your point. But I thought it was meant to reference the year and date it was exported, which in this case is 2022 (updated), although first 2019. Let me know if you want me to change it to 2019 anyways.

**Review by gabrieltseng — APPROVED** [review]
LGTM!

**Pull request merged**